src/planners/rbt/RGBTConnect.cpp

```diff
@@ -97,20 +97,20 @@ std::tuple<base::State::Status, std::shared_ptr<std::vector<std::shared_ptr<base
 {
     float d_c = ss->computeDistance(q);
 	std::shared_ptr<base::State> q_new = q;
```

Comment:
Use constructor, not assignment.

Reply:
Yes, it is used later in the loop: 
`std::shared_ptr<base::State> q_temp = ss->getNewState(q_new);`
thus state `q` will not be disrupted.